bigchaindb/tendermint/core.py

```diff
@@ -1,11 +1,12 @@
 """This module contains all the goodness to integrate BigchainDB
 with Tendermint."""
 
-
 from abci import BaseApplication, Result
+from abci.types_pb2 import ResponseEndBlock, ResponseInfo
```

Comment:
You can import everything from `abci`. As you can see from their [`abci/__init__.py`](https://github.com/davebryson/py-abci/blob/master/abci/__init__.py) they export all the useful classes.

Reply:
I can see`ResponseInfo` is exported but `ResponseEndBlock` is not exported there.